### PR TITLE
feat(ubuntu16): add ubuntu16 images

### PR DESCRIPTION
**Pull request by frickjack**
packer configs for ubuntu16_base and
ubuntu16_client

### CODE REVIEW

File: images/ubuntu16_base.json [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  "builders": [
+    {
+      "type": "amazon-ebs",
+      "access_key": "{{user `aws_access_key`}}",
+      "secret_key": "{{user `aws_secret_key`}}",
+      "region": "{{user `aws_region`}}",
+      "source_ami": "ami-da05a4a0",
+      "instance_type": "{{user `aws_instance_type`}}",
+      "ssh_username": "ubuntu",
+      "ami_name": "ubuntu16-docker-ks-crypted {{timestamp}}",
```
Comment (philloooo) [inline images/ubuntu16_base.json]: rm `ks`

File: images/ubuntu16_client.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    {
+      "type": "amazon-ebs",
+      "access_key": "{{user `aws_access_key`}}",
+      "secret_key": "{{user `aws_secret_key`}}",
+      "region": "{{user `aws_region`}}",
+      "source_ami": "{{user `ub16_source_ami`}}",
+      "instance_type": "{{user `aws_instance_type`}}",
+      "ssh_username": "ubuntu",
+      "encrypt_boot": true,
+      "ami_name": "ubuntu16-client-ks {{timestamp}}"
```
Comment (philloooo) [inline images/ubuntu16_client.json]: rm `ks`

File: images/ubuntu16_base.json (hunk 1 above)
Comment (frickjack) [reply images/ubuntu16_base.json]: done! :-)

File: images/ubuntu16_client.json (hunk 2 above)
Comment (frickjack) [reply images/ubuntu16_client.json]: done! :-)

**Pull request merged**